src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -304,6 +322,8 @@ public PreferenceSettings()
             ShowDetailedLayout = true;
             NamespacesToExcludeFromLibrary = new List<string>();
 
+            PythonTemplateFilePath = new PathManager(new PathManagerParams()).PythonTemplateFilePath;
```

Comment:
I've not investigated far enough to know, but do you actually need to create a new path manager just to pull this property out, don't we create a pathmanager somewhere else during dynamoModel initialization?